scanner/src/main/kotlin/LocalScanner.kt

```diff
@@ -138,7 +138,7 @@ abstract class LocalScanner(config: ScannerConfiguration) : Scanner(config) {
             val result = try {
                 log.info { "Starting scan of '${pkg.id}' (${index + 1}/${packages.size})." }
 
-                scanPackage(scannerDetails, pkg, outputDirectory, downloadDirectory)
+                scanPackage(scannerDetails, pkg, outputDirectory, downloadDirectory).map { it.copy(rawResult = null) }
```

Comment:
I believe this deserves a code comment. Some like "Remove the now unneeded reference to rawResult here to allow garbage collection to clean it up already now".

Reply:
All done.